### ISSUE TITLE
add tag to released versions

### ISSUE THREAD

**Issue by micheleorselli** [ideatosrl/Idephix#74]
idephix version is identified by the commit hash eg:

```
Idephix version d123fb7faa23338918bd84f6c0aceff51ed72ced released 2016-07-13 09:16:31
```

since now we are tagging releases we can also add this reference using the placeholder `git-tag`

**Comment by ftassi** [comment]
What about having just the release number (eg 0.12.0) ? Do we need the hash of the commit? 
I'm thinking of a selfupdate command that will be able to jump from a tagged version to another tagged version. Maybe we could also avoid major versioni switching (at least without some sort of --force option).

What do you think?

**Comment by micheleorselli** [comment]
well the initial scope of this issue is just about displaying this kind of information, not using it in any way, but once we have integrated it why not?

**Comment by ftassi** [comment]
Closed by f62da57bd20c7efa8aafbe7eb20dfb6c902450d0
